openedx/core/djangoapps/team_api/serializers.py

```diff
@@ -0,0 +1,52 @@
+from django.contrib.auth.models import User
+from rest_framework import serializers
+from openedx.core.lib.api.fields import ExpandableField
+from .models import CourseTeam, CourseTeamMembership
+from ..user_api.serializers import UserSerializer
+
+
+class CollapsedUserSerializer(serializers.HyperlinkedModelSerializer):
```

Comment:
I'll go ahead and add more `read_only` declarations. It doesn't affect behavior right now because only the CourseTeamSerializer is used for deserialization, but it would be good to have for consistency.